packages/gatsby/src/utils/feedback.ts

```diff
@@ -32,7 +32,7 @@ const randomChanceToBeTrue = (): boolean => {
       // One quarter year in days (roughly)
       (30 * 3)
   )
-  const randomNumberWithinQuarter = randomNumber * currentQuarter
+  const randomNumberWithinQuarter = randomNumber + 30 * 3 * (currentQuarter - 1)
```

Comment:
Hmm that's some fuzzy wuzzy code (the existing code I mean). I think your fix is correct.

`currentQuarter` is `1, 2, 3, 4` (offset 1). So it generates the offset `30 * 3 * (currentQuarter - 1)` and then adds a random number `0 - 90` to it to find a random day in the current quarter.